tests/homepage/addonList.test.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { expect } from 'chai';
+import { mount, shallow } from 'enzyme';
+import  { AddonList }  from '../../app/js/components/homePage/manageApps/addonList';
+
+
+describe('<AddonList />', () => {
+    const appList = [{
+        "version": "1.0.0",
+        "name": "CohortBuilder",
+        "description": "modern refresh of the OpenMRS Cohort Builder tool",
+        "icons": {
+            "16": null,
+            "48": "/img/omrs-button.png",
+            "128": null
+        },
+        "developer": {
+            "url": "https://github.com/openmrs/openmrs-owa-cohortbuilder",
+            "name": "andela-odaniel, wanjikum, andela-jomadoye, andela-pupendo, kingisaac95",
```

Comment:
Have all these devs contributed to this?

Reply:
No, it's for the Cohort builder. To test that the `addonList` compnent renders what comes from the API, I got an object from the API response and I'm using it to test that the component does what it should do - rendering the apps from the API